spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,output:tier::int,output:scalerank::int
+national_park;park or protected land;battlefield,!United States Forest Service,>=3,>=300000000,1,3
+national_park;park or protected land;battlefield,!United States Forest Service,>=4,>=300000000,1,4
+national_park;park or protected land;battlefield,!United States Forest Service,>=5,>=150000000,1,5
+national_park;park or protected land;battlefield,!United States Forest Service,>=6,>=150000000,1,6
+national_park;park or protected land;battlefield,!United States Forest Service,>=7,>=100000000,1,7
+national_park;park or protected land;battlefield,!United States Forest Service,>=8,>=10000000,1,8
+national_park;park or protected land;battlefield,!United States Forest Service,>=9,>=5000000,1,9
+national_park;park or protected land;battlefield,!United States Forest Service,>=10,>=1000000,1,10
+national_park;park or protected land;battlefield,!United States Forest Service,>=11,>=500000,1,11
+national_park;park or protected land;battlefield,!United States Forest Service,>=12,>=500000,1,12
+national_park;park or protected land;battlefield,!United States Forest Service,>=13,>=100000,1,13
+national_park;park or protected land;battlefield,!United States Forest Service,>=14,>=50000,1,14
+national_park;park or protected land;battlefield,!United States Forest Service,>=15,>=2000,1,15
+national_park;park or protected land;battlefield,!United States Forest Service,>=16,+,1,16
+national_park;park or protected land,!United States National Park Service,>=4,>=1000000000,2,4
```

Comment:
This is the same thing as saying:

```
filter:
  all:
    - $zoom: { min: 4 }
    - kind: [park, national_park]
    - not:
      all:
        - operator: [ "United States National Park Service", "United States Forest Service" ]
        - protect_class: ['2','3','5','6']
```

(because `not(A) or not(B)` is the same thing as `not(A and B)`)

This means it only affects features which have _both_ `operator` in the set above and `protect_class` in the set above. I find that counter-intuitive, so I wanted to check that's actually the logic you intended.